deploy/ansible/roles/asset-registry/templates/docker-compose.yml.j2

```diff
@@ -0,0 +1,34 @@
+#jinja2: trim_blocks:True, lstrip_blocks:True
+version: "3"
+services:
+  mongodb:
+    image: mongo:latest
+    container_name: mongodb
+    restart: unless-stopped
+    expose:
+      - 27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: "{{ mongoDBUser }}"
+      MONGO_INITDB_ROOT_PASSWORD: "{{ mongoDBPassword }}"
+    volumes:
+      - ./mongodb_data:/data/db
+    networks:
+      - back-tier
+
+  asset-registry:
+    image: angelocapossele/asset-registry
+    command: >
+      --basicAuthUser=IOTAisCool
+      --basicAuthPassword=WeAreThereforeCool
```

Comment:
These should be secrets

Reply:
Already fixed that just forgot to push to Github